torch/onnx/symbolic.py

```diff
@@ -868,14 +868,17 @@ def topk(g, self, k, dim, largest, sorted, out=None):
     return g.op("TopK", self, k_i=k, axis_i=dim, outputs=2)
 
 
-@parse_args('v', 'is')
```

Comment:
@apaszke @jamesr66a please advise whether removing this will break some jit stuffs that I'm not aware of.